serializers/src/OBJ/objSerializer.ts

```diff
@@ -16,9 +16,10 @@ export class OBJExport {
      * @param materials defines if materials should be exported
      * @param matlibname defines the name of the associated mtl file
      * @param globalposition defines if the exported positions are globals or local to the exported mesh
+     * @param sceneUsesRightHandedSystem defines if the babylon scene is right handed, in this case the exported obj X vertices won't be inverted 
      * @returns the OBJ content
      */
-    public static OBJ(mesh: Mesh[], materials?: boolean, matlibname?: string, globalposition?: boolean): string {
+    public static OBJ(mesh: Mesh[], materials?: boolean, matlibname?: string, globalposition?: boolean, sceneUsesRightHandedSystem?: boolean): string {
```

Comment:
`useRightHandedSystem` would be great. This is what we use throughout the framework

Reply:
I had named it `useRightHandedSystem` firstly but then changed as I think this can cause confusion as it could mean that we want the export to use a rightHandedSystem.

It is not the case, by default we want to convert left handed to right handed and the flag means : "hey I use rightHandedSystem in my scene, please keep the export as it is rightHanded"

Not sure if it makes sense 😉

Reply:
yeah, i get it. Naming is always a fun part of development.

I am fine with your current naming :-). As long as the documentation is clear, it is perfectly fine.